pkg/io/opener.go

```diff
@@ -184,25 +185,38 @@ func (o *opener) Reader(ctx context.Context, path string) (io.ReadCloser, error)
 }
 
 // Writer returns a writer that overwrites the path.
-func (o *opener) Writer(ctx context.Context, path string) (io.WriteCloser, error) {
-	if strings.HasPrefix(path, "gs://") {
-		g, err := o.openGCS(path)
+func (o *opener) Writer(ctx context.Context, p string, opts ...WriterOption) (io.WriteCloser, error) {
+	if strings.HasPrefix(p, "gs://") {
+		g, err := o.openGCS(p)
 		if err != nil {
 			return nil, fmt.Errorf("bad gcs path: %v", err)
 		}
 		if g != nil {
```

Comment:
This check shouldn't be needed, `openGCS` should either return an error or an `*ObjectHandle`

Reply:
Yup